bin/GCStress/GCStress.vcxproj

```diff
@@ -33,7 +33,7 @@
         $(ChakraCommonLinkDependencies);
         Advapi32.lib;
         %(AdditionalDependencies)</AdditionalDependencies>
-      <AdditionalLibraryDirectories>$(OutDir);%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
+      <AdditionalLibraryDirectories>$(OutDir);%(AdditionalLibraryDirectories);$(SdkLibPath)</AdditionalLibraryDirectories>
```

Comment:
Curious, what are the dependencies that require this path?

Reply:
Somehow for ARM64 Advapi32.lib was not picked from correct location.